src/ui/frames.c

```diff
@@ -1525,7 +1523,7 @@ meta_frames_button_press_event (GtkWidget      *widget,
           break;
         default:
           g_assert_not_reached ();
-          break;
```

Comment:
Why are you removing `break`?

Reply:
default don't require break

Reply:
It does not, but it is not reason to remove it, is it? I am with those who think you should always add `break` - https://softwareengineering.stackexchange.com/questions/201777/break-on-default-case-in-switch